pyartcd/pyartcd/pipelines/images_health.py

```
@@ -106,6 +118,113 @@ async def get_rebase_failures(self, version: str):
+            project="ART",
+            issue_type="Task",
+            summary=f"Image build failure: {image_name} ({group})",
+            description=description,
+            labels=labels,
+            components=["daily-ops"],
+        )
+        _LOGGER.info("Created ticket %s for %s (%s)", ticket.key, image_name, group)
+
+        try:
+            ticket.update(fields={"customfield_12320040": {"value": "Incidents & Support"}})
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)